### PR TITLE
DBTP-642 Missing log group name YAML patch override

### PR DESCRIPTION
**Pull request by gabelton**
Once a `subscription-filter` env addon has been created, each service in the app will need the log group name override file to be created. Although the override lives at svc level, it relies on an env addon existing and, therefore, it makes more sense to create the override as part of the make-addons command.

### CODE REVIEW

File: tests/copilot_helper/test_command_bootstrap.py [hunk 1]
```diff
@@ -115,6 +115,8 @@ def test_make_config(tmp_path):
     with open(str(tmp_path / "copilot/test-service/manifest.yml")) as service:
         assert service.read() == test_service_manifest
 
+    assert os.path.exists(str(tmp_path / "copilot/test-service/addons"))
```
Comment (gabelton) [inline tests/copilot_helper/test_command_bootstrap.py:118]: This test didn't make any assertions about the addons or overrides folders being created

**Comment by gabelton** [comment]
Think would also solve https://uktrade.atlassian.net/browse/DBTP-631

File: dbt_copilot_helper/commands/copilot.py [hunk 2]
```diff
@@ -186,6 +194,11 @@ def make_addons(directory="."):
     _cleanup_old_files(config, output_dir, env_addons_path)
     custom_resources = _get_custom_resources()
 
+    svc_names = list_copilot_local_services()
+    base_path = Path(directory)
+    for svc_name in svc_names:
+        _generate_svc_overrides(base_path, templates, svc_name)
```
Comment (WillGibson) [inline dbt_copilot_helper/commands/copilot.py:200]: Could this be called from `_generate_service_addons()`?

File: tests/copilot_helper/fixtures/make_addons/expected/web/overrides/cfn.patches.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+- op: replace
+  path: /Resources/LogGroup/Properties/LogGroupName
+  value: !Sub '/copilot/${AppName}/${EnvName}/${WorkloadName}'
+
```
Comment (WillGibson) [inline tests/copilot_helper/fixtures/make_addons/expected/web/overrides/cfn.patches.yml:4]:
```suggestion
```

File: dbt_copilot_helper/commands/copilot.py (hunk 2 above)
Comment (gabelton) [reply dbt_copilot_helper/commands/copilot.py:200]: I did look at that option, but I think we will miss some services, unless they happen to be defined under an addon in addons.yml. e.g.

```
dw-s3-bucket:
  type: s3
  services:
    - "s3proxy"
    - "beat"
    - "web"
    - "worker"
```
Not all applications will have a setup like DW

Comment (WillGibson) [reply dbt_copilot_helper/commands/copilot.py:200]: Oh yeah, I think I slightly misunderstood what was going on down below.

**Review by WillGibson — APPROVED** [review]

**Review by antroy-madetech — APPROVED** [review]

**Pull request merged**